tools/workspace/voxelized_geometry_tools/repository.bzl

```diff
@@ -0,0 +1,19 @@
+# -*- mode: python -*-
+# vi: set ft=python :
+
+load(
+    "@drake//tools/workspace:github.bzl",
+    "github_archive",
+)
+
+def voxelized_geometry_tools_repository(
+        name,
+        mirrors = None):
+    github_archive(
+        name = name,
+        repository = "calderpg/voxelized_geometry_tools",
```

Comment:
This will change back to the `ToyotaResearchInstitute` repo once the relevant [PR](https://github.com/calderpg/voxelized_geometry_tools/pull/9) (waiting on ROS2 testing) lands.